docs/Panorama/event-system.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: Event System
+parent: Panorama
+nav_order: 100
+---
+
+# Event System
+Panorama uses an event system to communicate between the engine and panels but also between panels. They can be defined and subscribed to via Javascript.
+
+## Defining Events
+```
+**TODO** What is the difference between these two functions
+```
+You can define your own global events within the ``scripts/util/event-definition.js`` file by calling the ``$.DefineEvent`` and ``$.DefinePanelEvent`` functions.
```

Comment:
I would keep panel events out of this, since they aren't really referenced anywhere else here and might just be confusing